pyproject.toml

```diff
@@ -100,49 +100,674 @@ dynamic = ["version", "optional-dependencies", "dependencies"]
 #
 # all, all-core, all-dbs, devel-all, devel-ci
 #
-# END BUNDLE EXTRAS HERE
-#
-# The ``doc`` extras are not available in the released packages. They are only available when you install
-# Airflow from sources in ``editable`` installation - i.e. one that you are usually using to contribute to
-# Airflow. They provide tools needed when you want to build Airflow documentation (note that you also need
-# ``devel`` extras installed for airflow and providers in order to build documentation for airflow and
-# provider packages respectively). The ``doc`` package is enough to build regular documentation, where
-# ``doc_gen`` is needed to generate ER diagram we have describing our database.
-#
-# START DOC EXTRAS HERE
-#
-# doc, doc-gen
-#
-# END DOC EXTRAS HERE
-#
-# The `deprecated` extras are deprecated extras from Airflow 1 that will be removed in future versions.
-#
-# START DEPRECATED EXTRAS HERE
-#
-# atlas, aws, azure, cassandra, crypto, druid, gcp, gcp-api, hdfs, hive, kubernetes, mssql, pinot, s3,
-# spark, webhdfs, winrm
-#
-# END DEPRECATED EXTRAS HERE
-#
-# !!!!!! Those provuders are defined in the `airflow/providers/<provider>/provider.yaml` files  !!!!!!!
-#
-# Those extras are available as regular Airflow extras, they install provider packages in standard builds
-# or dependencies that are necessary to enable the feature in editable build.
-# START PROVIDER EXTRAS HERE
-#
-# airbyte, alibaba, amazon, apache.beam, apache.cassandra, apache.drill, apache.druid, apache.flink,
-# apache.hdfs, apache.hive, apache.impala, apache.kafka, apache.kylin, apache.livy, apache.pig,
-# apache.pinot, apache.spark, apprise, arangodb, asana, atlassian.jira, celery, cloudant,
-# cncf.kubernetes, cohere, common.io, common.sql, databricks, datadog, dbt.cloud, dingding, discord,
-# docker, elasticsearch, exasol, fab, facebook, ftp, github, google, grpc, hashicorp, http, imap,
-# influxdb, jdbc, jenkins, microsoft.azure, microsoft.mssql, microsoft.psrp, microsoft.winrm, mongo,
-# mysql, neo4j, odbc, openai, openfaas, openlineage, opensearch, opsgenie, oracle, pagerduty,
-# papermill, pgvector, pinecone, postgres, presto, qdrant, redis, salesforce, samba, segment,
-# sendgrid, sftp, singularity, slack, smtp, snowflake, sqlite, ssh, tableau, tabular, telegram,
-# teradata, trino, vertica, weaviate, yandex, zendesk
-#
-# END PROVIDER EXTRAS HERE
-
+# PLEASE DO NOT MODIFY THIS SECTION MANUALLY. IT WILL BE OVERWRITTEN BY PRE-COMMIT !!
```

Comment:
Bad merge here. It should be regenerated after the recent change for dynamic properties in pyproject toml. Copying the current version from main and re-running pre-commit should fix it.